Gemfile

```diff
@@ -9,7 +9,7 @@ if RUBY_VERSION <= '1.8.7'
 end
 
 if RUBY_VERSION >= '2.0'
-  gem 'rubocop'
+  gem 'rubocop', '0.52.1'
```

Comment:
https://travis-ci.org/theforeman/foreman_maintain/jobs/350283391
I think we can lock it to `<= 0.52.1` ?

Reply:
While in general I agree with `<=`, rubocop is fairly specific as once you configure some cop that has not been defined in previous versions, it starts failing with older versions. Therefore I preffer matching against a specific version